modules/oadp-restore-single-vm-from-multiple-vm-backup.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-restore-single-vm-from-multiple-vm-backup_{context}"]
+= Restoring a single VM from a backup of many VMs
+
+If you have a backup containing many virtual machines (VMs), and you want to restore only one VM, you can use the `LabelSelectors` section in the `Restore` CR to select the VM to restore. To ensure that the Persistent Volume Claim (PVC) attached to the VM is correctly restored, and the restored VM is not stuck in a `Provisioning` status, use both the `app: <vm_name>` and the `kubevirt.io/created-by` labels. To match the `kubevirt.io/created-by` label, you can use the UID of the VM.
```

Comment:
I think if you backup with  matchLabels , you do not need the created by
but would like to verify